### DISCUSSION TITLE
Add a button next to Reply and Forward buttons on MessageView

### DISCUSSION THREAD

**Discussion by vood** [InboxSDK/InboxSDK#1113] [Q&A] [answered]
I'm looking for an option to add my custom buttons next to the standard Reply and Forward buttons (see the screenshot). Any suggestion on how I can do that? 

<img width="325" alt="Screenshot 2024-01-26 at 1 46 57 PM" src="https://github.com/InboxSDK/InboxSDK/assets/420428/b3bab921-e2ce-43f3-9f84-405b0fe22440">

**Comment by aleemstreak** [comment, marked as answer]
We don't officially support this, but we're looking to add this functionality. We'd be open to PRs

**Comment by vood** [comment]
Thanks, I ended up building it myself for writingmate.ai

[image: Screenshot 2024-01-26 at 2.37.45 PM.png]

On Fri, Jan 26, 2024 at 2:31 PM aleemstreak ***@***.***>
wrote:

> We don't officially support this, but we're looking to add this
> functionality. We'd be open to PRs
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/InboxSDK/InboxSDK/discussions/1113#discussioncomment-8262153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADGUTH55GKNMKGHQYYHFQTYQQVCVAVCNFSM6AAAAABCMYRFFOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENRSGE2TG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Best regards

Artem Vysotsky
e: ***@***.***
t: +1 650 931 5382

**Reply by aleemstreak** [reply]
would be great if you contributed back!
